gateserver/utils/structparse.py

```diff
@@ -0,0 +1,38 @@
+from collections import namedtuple
+from struct import Struct
+from enum import Enum
+
+STRUCT_FORMAT = '<' # little-endian, no alignment (i.e. packed)
```

Comment:
Hmm, tak mozno nieco ine. Ale STRUCT_FORMAT znie, akoby toto bol cely format string (a pritom je to len prvy znak).